src/fides/api/models/privacy_preference.py

```diff
@@ -41,12 +50,19 @@ class ConsentMethod(Enum):
 
 
 class ServingComponent(Enum):
-    # Not at the db level due to being subject to change.
-    # Only add here, do not remove
-    overlay = "overlay"
+    """
+    This differs from component type because we want to record exactly
+    where consent was served.
+
+    Not at the db level due to being subject to change.
+    Only add here, do not remove
+    """
+
+    overlay = "overlay"  # DEPRECATED. DO NOT REMOVE.
     banner = "banner"
+    modal = "modal"
```

Comment:
This previous Enum already had more granular options than ComponentType because it was used to communicate more specific detail about where a Notice was served - adding the modal type here.  I'd think going forward, FE would specify banner OR modal OR TCF overlay OR privacy center